tasks/DependencyCheck/task/index.ts

```diff
@@ -17,65 +20,73 @@ async function run(): Promise<void> {
     tl.debug(`Setting resource path to ${taskManifestPath}`);
     tl.setResourcePath(taskManifestPath);
 
-    const workspaceId: string = tl.getInput('workspaceId', true) as string;
-    const sharedKey: string = tl.getInput('sharedKey', true) as string;
-    const enableSelfHostedDatabase: boolean = tl.getBoolInput('enableSelfHostedDatabase', true);
-    const databaseEndpoint: string = tl.getInput('databaseEndpoint', (enableSelfHostedDatabase)) as string;
-    const scanPath: string = tl.getInput('scanPath', true) as string;
-    const excludedScanPathPatterns: string = tl.getInput('excludedScanPathPatterns', true) as string;
-
-    let repositoryName = (tl.getVariable('Build.Repository.Name'))?.split('/')[1];
-    let branchName = tl.getVariable('Build.SourceBranchName');
-    let buildName = tl.getVariable('Build.DefinitionName');
-    let buildNumber = tl.getVariable('Build.BuildNumber');
-    let commitId = tl.getVariable('Build.SourceVersion');
-
-    const la: ILogAnalyticsClient = new LogAnalyticsClient(
-      workspaceId,
-      sharedKey,
-    );
+    const enableVulnerabilityFilesMaintenance: boolean = tl.getBoolInput('enableVulnerabilityFilesMaintenance', true);
+    const writeStorageAccountContainerSasUri: string = tl.getInput('writeStorageAccountContainerSasUri', false) as string;
+    const workspaceId: string = tl.getInput('workspaceId', false) as string;
```

Comment:
my preference would be for a more explicit name as you've done with the SasUri above, eg logAnalyticsWorkspaceId, it's a preference though, don't think there's a right or wrong way

Reply:
though as they have full descriptions in task.json that's nitpicking

Reply:
Updated task parameter and variable names: workspaceId to logAnalyticsWorkspaceId, sharedKey to logAnalyticsWorkspaceKey